prototypes/styletransfer/model.py

```diff
@@ -160,15 +218,15 @@ def run_style_transfer(
         )
 
         # Create a nice config
-        loss_weights = (style_weight, content_weight)
+        loss_weights = LossWeights(style=style_weight, content=content_weight)
 
         # Compute the content2style ratio to balance losses
         c2s = self._estimate_content2weight(
             content_img, style_img, loss_weights, init_image
         )
 
         # update weights
-        loss_weights = (1.0, c2s)
+        loss_weights = LossWeights(style=1.0, content=c2s)
```

Comment:
This one should be `loss_weights = LossWeights(style=c2s, content=1.0)` according to what discussed here #38 
Anyway, we might want to keep the incorrect implementation as it works a lot better in the end. Not sure.

Reply:
I commented it out because of the loss, I add an issue to tune the weights now that we have 3 losses